.github/workflows/ci.yaml

```diff
@@ -0,0 +1,59 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: CI
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  e2e-kind:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bazel: ["4.0.0", "3.7.2"]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Bazel
+        uses: abhinavsingh/setup-bazel@v3
+        with:
+          version: ${{ matrix.bazel }}
+
+      - name: End-to-end (kind)
+        run: make test/e2e/kind
+
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bazel: ["4.0.0", "3.7.2"]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Bazel
+        uses: abhinavsingh/setup-bazel@v3
+        with:
+          version: ${{ matrix.bazel }}
+
+      - name: Tests
+        run: bazel test //apis/... //pkg/... //hack/... --test_output=streamed --test_output=errors
```

Comment:
Should we run the make target? That way we are all running the same thing

Reply:
I copied it from TeamCity. Looks like `test/all` is missing `//hack/...`. Shall we add it or use another target? Or multiple targets?

Reply:
Add it to that target

Reply:
Done